uptane-standard.md

```diff
@@ -846,11 +845,11 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 1. Load and verify the current time or the most recent securely attested time.
 1. Download and check the Root metadata file from the Director repository, following the procedure in {{check_root}}.
 1. Download and check the Timestamp metadata file from the Director repository, following the procedure in {{check_timestamp}}.
-1. Download and check the Snapshot metadata file from the Director repository, following the procedure in {{check_snapshot}}.
+1. Check the previously downloaded Snapshot metadata file from the Directory repository (if available). If the hashes and version number of that file match the hashes and version number listed in the new Timestamp metadata, there are no new updates and the verification process MAY be stopped and considered complete. Otherwise, download and check the Snapshot metadata file from the Director repository, following the procedure in {{check_snapshot}}.
 1. Download and check the Targets metadata file from the Director repository, following the procedure in {{check_targets}}.
-1. Download and check the Root metadata file from the Image repository, following the procedure in {{check_root}}.
+1. If the Targets metadata from the Directory repository indicates that there are no new targets that are not already currently installed, the verification process MAY be stopped and considered complete. Otherwise, download and check the Root metadata file from the Image repository, following the procedure in {{check_root}}.
 1. Download and check the Timestamp metadata file from the Image repository, following the procedure in {{check_timestamp}}.
-1. Download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
+1. Check the previously downloaded Snapshot metadata file from the Image repository (if available). If the hashes and version number of that file match the hashes and version number listed in the new Timestamp metadata, the ECU MAY skip to step 10. Otherwise, download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
```

Comment:
```suggestion
1. Check the previously downloaded Snapshot metadata file from the Image repository (if available). If the hashes and version number of that file match the hashes and version number listed in the new Timestamp metadata, the ECU MAY skip to the last step. Otherwise, download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
```